rectools/models/nn/transformer_base.py

```diff
@@ -354,6 +396,8 @@ def _init_lightning_model(
             verbose=self.verbose,
             train_loss_name=self.train_loss_name,
             val_loss_name=self.val_loss_name,
+            adam_betas=(0.9, 0.98),
```

Comment:
Linters now don't allow to skip kwargs, otherwise they think that arguments that come from   `**self._get_kwargs(self.lightning_module_kwargs)` are the ones that were skipped (like `adam_betas` if we didn't specify it.)

Reply:
ok